src/bbs2gh/Handlers/GenerateScriptCommandHandler.cs

```diff
@@ -62,13 +63,16 @@ private async Task<string> GenerateScript(GenerateScriptCommandArgs args)
         content.AppendLine(VersionComment);
         content.AppendLine(EXEC_FUNCTION_BLOCK);
 
-        var projects = await _bbsApi.GetProjects();
-        foreach (var (_, projectKey, projectName) in projects)
+        var projects = args.BbsProjectkey.HasValue()
+            ? new List<string>() { args.BbsProjectkey }
+            : (await _bbsApi.GetProjects()).Select(x => x.Key);
+
+        foreach (var projectKey in projects)
         {
-            _log.LogInformation($"Project: {projectName}");
+            _log.LogInformation($"Project: {projectKey}");
```

Comment:
I think displaying the project name is more user friendly in the logs or maybe we might want to display both?

Reply:
I did this intentionally. I think most BBS users are used to using the project key to refer to things (it shows up all over the place in the BBS UI), but also if we want project name that means we need to do a BBS API call to get it when they provide the key as an arg.